### PR TITLE
Rename SecretConversation to Conversation

### PR DESCRIPTION
**Pull request by daria-github**
Closes issue #224.

### CODE REVIEW

File: xmtp/src/conversation.rs [hunk 1]
```diff
@@ -84,17 +84,17 @@ impl ListMessagesOptions {
     }
 }
 
-// I had to pick a name for this, and it seems like we are hovering around SecretConversation ATM
```
Comment (richardhuaaa) [inline xmtp/src/conversation.rs:87]: We can just remove this comment

**Review by richardhuaaa — APPROVED** [review]
🥳

**Review by nakajima — APPROVED** [review]
Nice, this was definitely something that confused me when I first came into this codebase.

**Review by neekolas — APPROVED** [review]
Congrats on your first Rust PR @daria-github 🍾

**Pull request merged**